Lib9c/Lib9c.csproj

```diff
@@ -16,6 +16,9 @@
   <ItemGroup>
     <PackageReference Include="DecimalMath.DecimalEx" Version="1.0.2" />
     <PackageReference Include="JetBrains.Annotations" Version="2021.1.0" />
+    <PackageReference Include="MessagePack" Version="2.3.85" />
+    <PackageReference Include="MessagePack.Annotations" Version="2.3.85" />
+    <PackageReference Include="MessagePackAnalyzer" Version="2.3.85" />
```

Comment:
How about excluding this dependency from output assembly like `Libplanet.Analyzer`? (see also. https://github.com/planetarium/lib9c/pull/742/files#diff-934feb0de25b514ecae56d97457a94b5fb36f0c7f997165bc8f34d1f9b840b4eR36)